DEPENDENCY_LICENSES.md

```diff
@@ -1,14 +1,4 @@
-Emissary-ingress Go code incorporates Free and Open Source software under the following licenses:
-
-* [2-clause BSD license](https://opensource.org/licenses/BSD-2-Clause)
-* [3-clause BSD license](https://opensource.org/licenses/BSD-3-Clause)
-* [Apache License 2.0](https://opensource.org/licenses/Apache-2.0)
-* [ISC license](https://opensource.org/licenses/ISC)
-* [MIT license](https://opensource.org/licenses/MIT)
-* [Mozilla Public License 2.0](https://opensource.org/licenses/MPL-2.0)
-
-
-Emissary-ingress Python code incorporates Free and Open Source software under the following licenses:
+Emissary-ingress incorporates Free and Open Source software under the following licenses:
 
 * [2-clause BSD license](https://opensource.org/licenses/BSD-2-Clause)
 * [3-clause BSD license](https://opensource.org/licenses/BSD-3-Clause)
```

Comment:
This is clearly wrong, because there's no longer any mention of the MPL

Reply:
Do you mean MPL 2.0? I see it there in line 8
https://github.com/emissary-ingress/emissary/blob/aosorio/check-forbidden-licenses/DEPENDENCY_LICENSES.md